README.md

```diff
@@ -34,12 +34,28 @@ To learn about the steps involved in building from source, see [CONTRIBUTING](./
 
 ## Usage
 
+In order to use the Pinecone CLI you will need to authenticate with Pinecone services. This can be done either with an API key, or using the `pinecone login` flow to authenticate with a Pinecone account and your browser and leverage an authentication token.
+
 ```shell
 pinecone --help
 
+# If you have a Pinecone API set in your environment as PINECONE_API_KEY you can begin working with the CLI
+pinecone index list
+
+# To set an API key manually, you can use the config command
+pinecone config set-api-key "YOUR_API_KEY"
+
+# Additionally, you can authenticate manually using the login command
```

Comment:
`pinecone login` takes you through the browser auth process, right? If so, I might change the wording from "authenticate manually" (since all of these are kinda manual by way of being in a CLI) to "authenticate through the browser" or something similar